launch/LMS1xx.launch

```diff
@@ -1,6 +1,13 @@
 <launch>
   <arg name="host" default="192.168.1.14" />
+  
+  <!-- start and end units are in degrees to keep with lms internal structure. They will be converted to rad in the node. -->
+  <arg name="start" default="-999.9" />
+  <arg name="end" default="999.9" />
```

Comment:
Hmm. Okay, in that case I think I'm going to pass on this change— I think it would be a source of confusion for users, since the expectation would be that such a parameter would configure the device, not simply swallow data in the driver.